modules/tools/migration/src/main/java/org/wso2/mb/migration/DBConnector.java

```diff
@@ -208,9 +196,7 @@ void insertBindings(List<Binding> bindings) throws SQLException {
             }
             preparedStatement.executeBatch();
         } catch (SQLException e) {
-            e.printStackTrace();
-        } finally {
-            conn.close();
+            logger.error("Error while inserting bindings", e);
```

Comment:
Hi,
Yes this is intentional. In insert bindings we haven't thrown the error because we are allowing the tool to run.